recipe/meta.yaml

```
@@ -61,7 +61,7 @@ outputs:
         - itsdangerous
         - jinja2
         - minio
-        - pydantic
+        - pydantic <2.0
```

Not that it'll be a problem here, but we usually pin strictly from alphas to avoid prereleases being accidentally included in the interval.

```suggestion
        - pydantic <2.0a0
```

This is a safer idiom in conda world where we don't have `--pre` flags (yet?).

---

good shout ✅ adding and this should be ready to merge